bliss/train.py

```diff
@@ -111,11 +118,18 @@ def setup_callbacks(cfg) -> Optional[ModelCheckpoint]:
             save_on_train_epoch_end=False,
             auto_insert_metric_name=False,
         )
-    else:
-        checkpoint_callback = None
     return checkpoint_callback
 
 
+def setup_early_stopping(cfg):
+    early_stopping = None
+    if cfg.training.enable_early_stopping:
+        early_stopping = EarlyStopping(
+            monitor="val/loss", mode="min", patience=cfg.training.patience
+        )
+    return early_stopping
+
```

Comment:
Ditto.

Reply:
Fixed by enabling early stopping in tests.